terraform/modules/baseline/variables.tf

```diff
@@ -151,6 +151,7 @@ variable "ec2_autoscaling_groups" {
     })
     instance = object({
       disable_api_termination      = bool
+      disable_api_stop             = bool
```

Comment:
Can we have this optional - so it doesn't break existing stuff

Reply:
updated